package.json

```diff
@@ -46,10 +47,13 @@
     "@rollup/plugin-node-resolve": "^13.1.1",
     "@scure/base": "^1.1.1",
     "commander": "^9.4.1",
+    "near-abi": "file:../near-abi-js",
```

Comment:
@itegulov 
Is it a separate crate in Rust? If yes, looks like we will need to move to the monorepo arch @ailisp.

Reply:
It is. Basically an unopinionated set of models that people can use to parse/work with ABIs: https://github.com/near/near-abi-rs. If you don't think it will be useful as its own library, I can move it here for sure.

Reply:
Since it's similarly organized as rust, as a separate looks good